kit/inputs/src/MaterialTextView/hooks/useClear.ts

```diff
@@ -9,7 +9,7 @@ export function useClear(
     return React.useCallback(
         function clear() {
             imperativeChangeText('');
```

Comment:
Does this call ensure that the same effect as `ref.current?.clear();` is applied? I just want to say if we need to remove the `ref.current?.clear();` line bellow?